openshift/release/release-notes.md

```diff
@@ -0,0 +1,169 @@
+# OpenShift Pipelines 1.4
+
+# Tekton Pipelines v0.22.0
+
+## Features
+N/A
+
+## Deprecation Notices
+N/A
+
+## Backwards incompatible changes
+N/A
+
+## Fixes
```

Comment:
@piyush-garg for all versions together, there are a lot of bug fixes, are all of them relevant to us and worth documenting?